### PR TITLE
Archive old documents, rewrite README.md

### PR DESCRIPTION
**Pull request by adzialocha**
This rewrite serves as a first step towards our new website / documentation / `README.md`.

* Archive all documents which are outdated
* Have a more general `README.md` for now

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,57 +1,107 @@
...
+* **Browser friendliness**: Lightweight clients that can easily be implemented for websites.
+* **Schemas**: Users suggest data schemas which can be installed on nodes, validating, storing and indexing the data for the network.
+* **Encryption**: Transport communication is end-to-end encrypted via SSB's Secret Handshake, data is encrypted for groups via the Messaging Layer Security (MLS) protocol.
+* **Integrity**: Data is signed and can only be created and updated by its original author.
+* **Deletion**: Data can be deleted without loosing verifiability and log integrity.
+* **Decentralization**: Networks consist of both federated or decentralised nodes.
+* **Partial replication**: Nodes do not need to download the whole log to verify them.
+* **Low-power electronics**: A bunch of laptops or single board computers need less electricity than a server farm.
+* **Identities**: People manage one or many identities across devices.
+* **Moderation**: Support hiding or removing content for users and groups.
```
Comment (cafca) [inline README.md:20]: Suggestion: Decentralised content moderation for users and groups.

The current phrasing sounds like we can just go and delete people's stuff which is kind of the opposite? :D

File: README.md [hunk 2]
```diff
@@ -1,57 +1,107 @@
...
+* **Local- & offline first**: Communicate without the global internet.
```
Comment (cafca) [inline README.md:21]: I would put this way higher on this list! Access to online services without reliable and performant internet infrastructure. Independence from the corporate cloud.

File: README.md [hunk 3]
```diff
@@ -1,57 +1,107 @@
...
-- **Offline first:** Every device has its own copy of all information, which lets p2panda work independent of an Internet connection, obliterates a cloud infrastructure and creates an ephemeral archive for every participant. p2panda clients create a peer-to-peer network to exchange and sync information.
-- **Open Protocol:** p2panda is an easy-to-extend protocol that records the connection of users, resources and events. You can create a serendipity bot that occasionally creates random meeting events for your festival goers, or a hardware display with a live update of the festival schedule, or an archive bot that makes available a permanent copy of the festival on the world wide web.
-- **Warmth:** Computers make it easy to get carried away by their rigidly structured ways. However, every computer also contains an undeniable spark of pure chaos. We want to capture that spark to ignite a campfire for you to gather around and get cosy.
+* **Browser friendliness**: Lightweight clients that can easily be implemented for websites.
+* **Schemas**: Users suggest data schemas which can be installed on nodes, validating, storing and indexing the data for the network.
```
[the remark this answers is not in the document]
Comment (cafca) [reply README.md:12]: Maybe add sth abt  indexing the data for the network.

File: README.md [hunk 4]
```diff
@@ -1,57 +1,107 @@
 # p2:panda_face:
 
-**Note June 2021: This document reflects where we started off, it shows our motivations and dreams. Since then, p2panda has grown to be more than a software for festivals. An update to this design document is coming, keep this in mind until then.**
+p2panda is a user-friendly peer-to-peer communications protocol for secure, energy-efficient, offline- and local-first web applications. We want this protocol to be a playful tool for people to hack, build, play, and experiment with.
```
Comment (sandreae) [inline README.md:3]: Is it "energy-efficient" ?

File: README.md [hunk 5]
```diff
@@ -1,57 +1,107 @@
...
 
-p2panda is a :panda_face: way to get together in self-organized festivals. 
+Messages in p2panda are signed, encrypted and published by clients using the [Bamboo](https://github.com/AljoschaMeyer/bamboo) append-only log data type which gets distributed over the network to other peers. p2panda allows for decentralised or federated network topologies or even hybrids of these and aims at running in web browsers without loosing its cryptographic features.
```
Comment (sandreae) [inline README.md:5]: "decentralised and federated"

File: README.md [hunk 6]
```diff
@@ -1,57 +1,107 @@
...
 
-A festival can be anything you want to plan with your friends, your circle, your collective, your commune – or people you have never met before: Give p2panda to your devices and create workshops, gatherings, initiatives, concerts or conferences using the computers and phones you already have – independent of any commercial infrastructure.
+> **p2panda is currently very much in prototyping and specification phase, our milestones and progress can be seen below. If you're interested in any of these topics please get in touch!**
```
Comment (sandreae) [inline README.md:7]: Link the "get in touch" link to the bottom get in touch section.

File: README.md [hunk 7]
```diff
@@ -1,57 +1,107 @@
...
 
-What is in it?
+## Topics we're interested in
```
Comment (sandreae) [inline README.md:9]: Should the topics be in alphabetical order?

File: README.md [hunk 8]
```diff
@@ -1,57 +1,107 @@
...
+</div>
 
-:eight_spoked_asterisk: [Code of Conduct](https://github.com/p2panda/design-document/blob/master/CODE_OF_CONDUCT.md)
+This led to a group of people interested in realising a protocol for p2p communication, which ultimately should serve as a tool to build applications, like a festival tool and more.
 
-:eight_spoked_asterisk: [Technical design](https://github.com/p2panda/design-document/blob/master/DESIGN.md) and implementation ideas
+In recent times activities have gathered around [Liebe Chaos Verein](https://liebechaos.org/), organizing a [p2p gathering](https://p2p-berlin.org/) and reading group in Berlin. We've been meeting almost every Monday since 2019 to hack on p2panda. Obviously we're still gunna organise another festival sometime :panda_face:
 
-:eight_spoked_asterisk: [Specification](https://github.com/p2panda/design-document/blob/master/spec) of the protocol (sloth draft)
+## Overview
```
Comment (sandreae) [inline README.md:41]: Find better title... what is this section?

Comment (sandreae) [reply README.md:41]: "Repositories" or "Project Structure"

Comment (sandreae) [reply README.md:41]: Intro paragraph needed.

File: README.md [hunk 9]
```diff
@@ -1,57 +1,107 @@
...
+* [ ] Automatic discovery via gossip protocol
+* [ ] Efficient point2point replication protocol
+* [ ] Schema backwards compatibility via lenses
+* [ ] p2panda specification 1.0 release :panda_face:
+* [ ] Automatic deletion of unused data
+* [ ] Automatic detection of forked logs
+* [ ] Filter and content moderation curated by users and groups
 
-:diamond_shape_with_a_dot_inside: [HOFFNUNG 3000](https://github.com/adzialocha/hoffnung3000) - Platform for self-curated festivals
+## How to contribute
```
Comment (sandreae) [inline README.md:92]: Need contributor guidelines and conditions.

File: README.md [hunk 10]
```diff
@@ -1,57 +1,107 @@
...
 
-:diamond_shape_with_a_dot_inside: [Shirokuma Cafe](https://en.wikipedia.org/wiki/Shirokuma_Cafe) - :panda_face:
+* Come to our weekly meetings, every Monday at 19:00 CET: https://wolke.liebechaos.org/call/tq6j9tuz
```
Comment (sandreae) [inline README.md:94]: Are the Monday meetings public? They probably need more coding if they are :rofl:

**Pull request merged**